<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="lib/favicon_biohazard.ico">

    <title>3DFlu - database and exploration</title>

    <link href="lib/bootstrap.min.css" rel="stylesheet">
    <link href="lib/sticky-footer-navbar.css" rel="stylesheet">

  </head>

  <body>

    <nav class="navbar navbar-default navbar-fixed-top">
      <div class="container">
        <div class="navbar-header">
          <a class="navbar-brand" href="#">3DFlu</a>
        </div>
        <div id="navbar" class="collapse navbar-collapse">
          <ul class="nav navbar-nav">
            <li><a href="index.html">Home</a></li>
            <li><a href="search.html">Search</a></li>
            <li><a href="exploration.html">Exploration</a></li>
            <li class="active"><a href="dataset.html">Dataset</a></li>
            <li><a href="authors.html">Authors</a></li>
          </ul>
        </div>
      </div>
    </nav>

    <div class="container">

      <p class="lead">Sources</p>
      <p>(TO FIX)</p>

<p>Data sources:</p><ul><li>Sequences</li></ul><ul style="list-style: none;"><li>Sequences of selected serotypes were extracted from the following databases:&nbsp;</li><li>Uniprot (<a href="http://www.uniprot.org/)"></a>http://www.uniprot.org/),</li><li>NCBI (<a href="http://www.ncbi.nlm.nih.gov/),README"></a>http://www.ncbi.nlm.nih.gov/),<a href='https://hackpad.com/WFzsvu2fFFF'/>README</a></li><li>GIS-AID (<a href="http://platform.gisaid.org/epi3/frontend#2f0a9a)"></a>http://platform.gisaid.org/epi3/frontend#2f0a9a),</li><li>Influenza Research Database (<a href="http://www.fludb.org/brc/home.spg?decorator=influenza)"></a>http://www.fludb.org/brc/home.spg?decorator=influenza),</li><li>EMBL-EPI (<a href="http://www.ebi.ac.uk/)"></a>http://www.ebi.ac.uk/)</li></ul><ul><li>Structures</li></ul><ul style="list-style: none;"><li>Crystal structures of Hemagglutinin proteins were downloaded from the Protein Data Bank (PDB) (<a href="http://www.rcsb.org)"></a>http://www.rcsb.org).</li></ul><ul><li>Metadata</li></ul><ul style="list-style: none;"><li>Furthermore for each entry, metadata information such the virus subtype, the year of the infection, the host species and the geographic location of the infected host are included. These information were obtained directly from the above mentioned databases. Incomplete data were supplemented with information available in literature.</li></ul><p>Programs:</p><ul><li>For homology modeling procedure the following tools were used:&nbsp;<ul><li>BLASTp (version 2.2.28+) (<a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi)"></a>http://blast.ncbi.nlm.nih.gov/Blast.cgi) was chosed to select the closest template meeting the following specific criteria: resolution value &lt; 2.6<b>Ã…</b>, free R-value &lt; 3.0.</li><li>Sequence to structure alignment was calculated using the PCMA (version 2.0) (<a href="http://prodata.swmed.edu/pcma/pcma.php)"></a>http://prodata.swmed.edu/pcma/pcma.php) software with default parameters.</li><li>Structures were build using MODELLER (version 9.15) (<a href="https://salilab.org/modeller/)"></a>https://salilab.org/modeller/). For each sequence 10 structures were calculated. MODELLER pseudo-energy criterium was applied in order to select the best candidate for each run.</li></ul></ul><ul><li>In order to calculate the similarity between sequence from both crystals and models, the following tool was used:&nbsp;<ul><li>The Python library pairwise2 (<a href="http://biopython.org/DIST/docs/api/Bio.pairwise2-module.html)"></a>http://biopython.org/DIST/docs/api/Bio.pairwise2-module.html) was used to calculate pairwise sequence alignments. The e-values obtained were used as sequence similarity measures.&nbsp;</li></ul></ul><ul><li>RMSD distances between protein structures from both crystals and models were calculated using the following tool:<ul><li>&nbsp;The TM-align method (<a href="http://zhanglab.ccmb.med.umich.edu/TM-align/)"></a>http://zhanglab.ccmb.med.umich.edu/TM-align/<a href='http://zhanglab.ccmb.med.umich.edu/TM-align/was'/>)</a><a href='http://zhanglab.ccmb.med.umich.edu/TM-align/)was'/> </a>was used to superimpose and calculate RMSD between respective entries.</li></ul></ul><ul><li>Electrostatic Potential for both structures and models was computed with:<ul><li>TM-align was used to perform the structural alignment both models and strucures&nbsp;</li><li>APBS (version 1.4.1) (<a href="http://www.poissonboltzmann.org)"></a>http://www.poissonboltzmann.org) was used to calculate the electrostatic potential grids of the above mentioned structures. Only structural region closed to the binding site was included in the calculations</li></ul></ul><ul><li>In order to calculate the ESP distances between protein structures from both crystals and models the following tools was used:<ul><li>Pipsa (version 3.1) (<a href="http://pipsa.eml.org/pipsa/)"></a>http://pipsa.eml.org/pipsa/<a href='http://pipsa.eml.org/pipsa/was'/>)</a><a href='http://pipsa.eml.org/pipsa/)was'/> </a>was used to compare electrostatic potential grids and generate similarity values.</li></ul></ul><ul><li>The protein&rsquo;s residues mobility was estimated using the following Python package:&nbsp;<ul><li>&nbsp;The Gaussian Network Model (GNM) (<a href="http://prody.csb.pitt.edu/tutorials/enm_analysis/gnm.html)"></a>http://prody.csb.pitt.edu/tutorials/enm_analysis/gnm.html) implemented in ProDY was applied for this purpose</li></ul></ul><ul><li>&nbsp;Secondary structure assignment was performed using:<ul><li>DSSP (<a href="http://www.cmbi.ru.nl/dssp.html)"></a>http://www.cmbi.ru.nl/dssp.html<a href='http://www.cmbi.ru.nl/dssp.html'/>)</a><a href='http://www.cmbi.ru.nl/dssp.html'/>&nbsp; </a>was&nbsp; used to assign secondary structure for proposed models</li></ul></ul><ul><li>Multiple Sequence Aligments (MSA) of all the entries within the database was generated using the following methods:<ul><li>PCMA (): generates MSA using protein sequences uniquely</li><li>3D-Coffee (<a href="http://www.tcoffee.org/Projects/expresso/)"></a>http://www.tcoffee.org/Projects/expresso/): generates structure-based MSA using both sequence-based and&nbsp; structural-based information</li></ul></ul><p></p><p>Data Structure</p><p></p><ul><li>File description<ul><li>root/<ul><li>models_template_mapping.csv: Contains the information about the template used for modeling a give sequence. The csv file contains two columns:&nbsp;<ul><li><b><i>p_id:</i></b> sequence used for modeling,&nbsp;</li><li><b><i>template_id:</i></b> chosen template</li></ul></li></ul></li><li>root/aligned_sequences/sequence_alignment:<ul><li>crystals_aln.csv: The file contains multiple sequence alignment for structures extracted from PDB. The csv file contains two columns:</li><ol><li><b><i>p_id</i></b>: PDB protein identifier</li><li><b><i>sequence</i></b>: sequence aligned to the rest of the entries</li></ol></ul><ul><li>models_aln.csv:&nbsp; The file contains multiple sequence alignment for the sequences selected for modeling. The csv file contains two columns:<ul><li><b><i>p_id</i></b>: GenBank Protein Accession identifier of sequence</li><li><b><i>sequence</i></b>: sequence aligned to the rest of the entries</li></ul></li></ul></li><li>root/aligned_sequences/structural_alignment:<ul><li>crystals_aln_str.csv: The file contains multiple sequence alignment for structures extracted from PDB. The csv file contains two columns:<ul><li><b><i>p_id</i></b>: PDB protein identifier</li><li><b><i>sequence</i></b>: sequence structurally aligned to the rest of the entries</li></ul></li><li>models_aln_str.csv:&nbsp; The file contains multiple sequence alignment for the sequences selected for modeling. The csv file contains two columns:</li><ol><li><b><i>p_id: </i></b>GenBank Protein Accession identifier of sequence</li><li><b>sequence</b>: sequence structurally aligned to the rest of the entries</li></ol></ul></ul><ul><li>root/distances/crystals/:<ul><li>crystals_esp.csv<ul><li><b><i>p_id:</i></b> identifier of the 1st protein</li><li><b><i>p_id2</i></b>: identifier of the 2nd protein</li><li><b><i>distance </i></b>: similarity value for compared proteins ESP maps</li></ul></li><li>crystals_rmsd.csv<ul><li><b><i>p_id:</i></b> identifier of the 1st protein</li><li><b><i>p_id2</i></b>: identifier of the 2nd protein</li><li><b><i>distance</i></b>: RMSD distance between protein structures</li></ul></li><li>crystals_seq_sim.csv<ul><li><b><i>p_id</i></b>: identifier of the 1st protein</li><li><b><i>p_id2</i></b>: identifier of the 2nd protein</li><li><b><i>distance</i></b>: normalized FASTA score between the pair of protein sequences (This are scores and not distances: higher the score, smaller would be the relative distance)</li></ul></li></ul></li><li>root/distances/models/:<ul><li>models_esp.csv<ul><li><b><i>p_id</i></b>: identifier of the 1st protein</li><li><b><i>p_id2</i></b>: identifier of the 2nd protein</li><li><b><i>distance</i></b>: similarity value for compared proteins ESP maps</li></ul></li><li>models_rmsd.csv<ul><li><b><i>p_id</i></b>: identifier of the 1st protein</li><li><b><i>p_id2</i></b>: identifier of the 2nd protein</li><li><b><i>distance</i></b>: RMSD distance between protein structures</li></ul></li><li>models_seq_sim.csv<ul><li><b><i>p_id</i></b>: identifier of the 1st protein</li><li><b><i>p_id2</i></b>: identifier of the 2nd protein</li><li><b><i>distance</i></b>: normalized FASTA score between the pair of protein sequences (This are scores and not distances: higher the score, smaller would be the relative distance)</li></ul></li></ul></li><li>root/metadata/<ul><li>crystals_metadata.csv:<ul><li><b><i>id</i></b>: PDB protein id (e.g. &rsquo;4cqv&rsquo;)</li><li><b><i>chain</i></b>: protein chain (e.g. &rsquo;A&rsquo;)</li><li><b><i>subtype</i></b>: HA subtype (e.g &rsquo;H5N1&rsquo;)</li><li><b><i>host:</i></b> HA host (e.g &rsquo;avian&rsquo;)</li><li><b><i>location</i></b>: location of sample collection (e.g. &rsquo;Turkey&rsquo;)</li><li><b><i>year: </i></b>year of sample collection (e.g. &rsquo;2005&rsquo;)</li></ul></li><li>models_metadata.csv:<ul><li><b><i>id</i></b>: gene id of the model (e.g. &rsquo;AAA43205&rsquo;)</li><li><b><i>chain</i></b>: protein chain (e.g. &rsquo;A&rsquo;)</li><li><b><i>subtype</i></b>: HA subtype (e.g &rsquo;H5N9&rsquo;)</li><li><b><i>host:</i></b> HA host (e.g &rsquo;avian&rsquo;)</li><li><b><i>location</i></b>: location of sample collection (e.g. &rsquo;Ontario&rsquo;)</li><li><b><i>year: </i></b>year of sample collection (e.g. 1966)</li></ul></li></ul></li><li>root/structures/crystals<ul><li>Here are located, as separate PDB files, all the structures of HA proteins downloaded from PDB database and filtered. The original PDB files underwent the following pre-processing:</li><ul style="list-style: none;"><li>1. In case of multi-chain structure (e.g. HA trimers) only single chain was kept.</li><li>2. The structures were purged of all the existing heteroatoms.</li><li>3. The C-term and N-term were trimmed after structural alignment in order to remove structurally variable regions.</li></ul></ul></ul><ul><li>root//structures/models<ul><li>Here are deposited, as separate PDB files. all the HA proteins models available in this database</li></ul></li><li>root/sequences/crystals<ul><li>crystals_pdb.fa<ul><li>In this file raw sequences of analyzed proteins were put in a fasta format. In this case the sequences have been directly extracted from the PDB structures.</li></ul></li><li>crystals.fa<ul><li>In this file raw sequences of analyzed proteins were put in a fasta format. In this case the sequences have been retrieved from uniprot and then cut in order to have the same length of sequences extracted from the PDB structures. This procedure allows to account all the AA within the selected intervals, including the internal AA that are missing in some PDB structures (e.g. 3s12).</li></ul></li></ul></li><li>root/sequences/models<ul><li>models.fa<ul><li>Tis file contains raw sequences of proteins for 3D structures were proposed. The sequences are in fasta format.</li></ul></ul></ul></ul>

    </div>

    <footer class="footer">
      <div class="container">
        <p class="text-muted"><b>3DFlu</b> - database and exploration of hemagglutinin proteins of influenza virus type A; developed at CeNT, University of Warsaw</p>
      </div>
    </footer>

  </body>
</html>